docs/logical_data_model.puml

```diff
@@ -137,6 +137,7 @@ class ActivityReportObjectives{
  arOrder : integer : 1
  closeSuspendContext : text
  closeSuspendReason : enum
+ objectiveCreatedHere : boolean
```

Comment:
Could we have a source column on the Objective instead?

Reply:
It needs to track to a specific report.